EventListener/TransformerEventListener.php

```diff
@@ -143,6 +143,12 @@ private function hasTransformer($object)
      */
     private function getTransformerAnnotation($object)
     {
+        $pattern = 'Proxies\\__CG__\\';
+        if (0 === strpos(get_class($object), $pattern)) {
+            $notProxiClassName = substr(get_class($object), strlen($pattern));
+            $object = new $notProxiClassName;
```

Comment:
`\Doctrine\Common\Util\ClassUtils::getRealClass(get_class($object))` would be more efficient :-)